dataactbroker/handlers/fileHandler.py

```diff
@@ -771,6 +771,7 @@ def check_detached_generation(job_id):
         """
         sess = GlobalDB.db().session
 
+        # TODO is this comment wrong or is the code wrong?
```

Comment:
Looks like the comment is wrong. This may also be changed in the D file generation so we'll check it out when we merge.